runtime/cudaq/algorithms/run.h

```diff
@@ -125,7 +126,7 @@ run(std::size_t shots, QuantumKernel &&kernel, ARGS &&...args) {
 #endif
   // Launch the kernel in the appropriate context.
   auto &platform = cudaq::get_platform();
-  std::string kernelName{cudaq::getKernelName(kernel)};
+  std::string kernelName{details::getKernelName(kernel)};
```

Comment:
Aside: weren't we going to converge on `detail` (without the `s`)? Might want to do that here?